### ISSUE TITLE
make installer manjaro compatible

### ISSUE THREAD

**Issue by discipolo** [blinkreaction/drude#91]
would be a shame if this were only compatible with ubuntu 14.04

**Comment by lmakarov** [comment]
I've been thinking about this and similar requests we've been receiving internally.

It will probably make more sense to write manual setup instructions for Linux.
There are just too many distributions out there to support and maintain an automated setup script for all of them.

**Comment by discipolo** [comment]
i totally agree that its not a good idea to attempt a multiplatform install script.

it should be made possible and clear though that this great project can be used with other distros than nubuntu.
I am looking forward to a [CONTRIBUTING.md](https://github.com/blog/1184-contributing-guidelines) file and will gladly create a [wiki page](https://github.com/blinkreaction/drude/wiki) with install instructions for arch based systems according to the instructions i find there.
